WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  InPersonPaymentsStripeAcountReview.swift
+//  WooCommerce
+//
+//  Created by Cesar Tardaguila on 2021-08-10.
+//  Copyright © 2021 Automattic. All rights reserved.
+//
+
+import SwiftUI
+
+struct InPersonPaymentsStripeAcountReviewView: View {
```

Comment:
This is a small nitpick that I missed in previous reviews, but it'd be good for consistency.

I'm not using the `View` suffix for SwiftUI views (even though I'm keeping it for the file name). None of the system views have it (`Text`, `Image`,...) so I do the same for the custom ones.

Reply:
Renamed. I will push an update to rename also some of the other pre-existing views.